charts/spire/templates/spire-server-namespace.yaml

```diff
@@ -1,14 +1,27 @@
-{{- if .Values.global.spire.namespaces.server.create }}
+{{- if or .Values.global.spire.namespaces.create .Values.global.spire.namespaces.server.create }}
+{{-   $labels := dict }}
+{{-   if and (dig "spire" "recommendations" "enabled" false .Values.global) (dig "spire" "recommendations" "namespacePSS" true .Values.global) }}
+{{-     $_ := set $labels "pod-security.kubernetes.io/warn" "restricted" }}
+{{-     $_ := set $labels "pod-security.kubernetes.io/audit" "restricted" }}
+{{-     $_ := set $labels "pod-security.kubernetes.io/enforce" "restricted" }}
```

Comment:
is it possible to break these out into yaml so its easier to read?

Reply:
Updated with them broken out.